### PR TITLE
Accept burgerservicenummer parameter

### PR DESCRIPTION
**Pull request by Vegter**
De BRP API is met het burgerservicenummer als path variabele gebouwd, echter deze dient ook als query parameter beschikbaar te zijn.

### CODE REVIEW

File: src/gobstuf/stuf/brp/base_request.py [hunk 1]
```diff
@@ -55,8 +55,6 @@ def _set_values(self, values: dict):
         :param values:
         :return:
         """
-        assert values.keys() == self.parameter_paths.keys()  # Should never fail
```
Comment (JKoetsier) [inline src/gobstuf/stuf/brp/base_request.py:58]: Why is this line removed?

Comment (JKoetsier) [reply src/gobstuf/stuf/brp/base_request.py:58]: Ah, I see now why it's removed. Shouldn't we just change this check instead of removing it?

Comment (Vegter) [reply src/gobstuf/stuf/brp/base_request.py:58]: Because every argument combination is already checked on receipt.
So the argument combination is already checked for validity
And not all parameters are required. Combination bsn requires 1 parameter, postcode, buisnummer 2 parameters. But not all have to be specified.
The assertion will fail because there are 3 possible parameters (bsn, postcode, huisnummer) but not all are available.
But please suggest another option if I have missed something.

Comment (Vegter) [reply src/gobstuf/stuf/brp/base_request.py:58]: Well, it has already been checked when the combination was checked so why check it twice?

Comment (JKoetsier) [reply src/gobstuf/stuf/brp/base_request.py:58]: Development errors. This class assumes (correctly, at this moment) that the calling class has checked the parameter combinations. That is some external knowledge that's verified by the assertion. You and I know at this moment how that interaction works, but it's easy to make a mistake there in the future. This assertion would fail at the first run that this precondition is broken. Yes, eventually the code would probably fail further down the line, but that is unpredictable. I think an assertion makes it clear as early as possible that a developer did something wrong (instead of generating a cryptic error message further down the line that eventually resolves to the same problem)

Comment (Vegter) [reply src/gobstuf/stuf/brp/base_request.py:58]: OK, point taken, I will update the assertion...
